src/Vts/MonteCarlo/DataStructures/SimulationInput.cs

```diff
@@ -39,10 +38,7 @@ public class SimulationInput
             TissueInput = tissueInput;
             DetectorInputs = detectorInputs;       
             // check if detectorInputs list is null and if so make empty
-            if (DetectorInputs == null)
-            {
-                DetectorInputs = new List<IDetectorInput>() {};
-            }
+            if (DetectorInputs == null)  DetectorInputs = new List<IDetectorInput> {};

```

Comment:
This is not a return so should it be on one line? Was this a cleanup suggestion?

Reply:
I saw this later in the cleanup and I think on one line is ok, I just had a question about whether we should have removed the ( ) or the { }

Reply:
Replaced "{}" with "()"